_data/menu_items.yml

```diff
@@ -1,18 +1,16 @@
 it:
   - title: Home
     url: /
-  - title: Modelli
-    url: /modelli
-  - title: Kit
-    url: /kit
   - title: Linee guida
     url: /linee-guida
+  - title: Kit
+    url: /kit
+  - title: Modelli
+    url: /modelli
+  - title: Piano attività
+    url: /piano-attivita
   - title: Notizie
     url: /notizie
-  - title: Progetti
-    url: /progetti
```

Comment:
Is this removal on purpose?

Reply:
Would be nice to remove — in the next release — also the 1st item "Home", it is pretty unuseful. What do you think @bfabio ?

Reply:
FYI @danielaiozzo